repl/lib/js/main.js

```diff
@@ -1,3 +1,6 @@
+import sanctuary from 'sanctuary';
+import ramdaFantasy from 'ramda-fantasy';
```

Comment:
We could use jsdelivr to serve Sanctuary and RF. Unfortunately, this means work:

1. Neither lib is on jsdelivr now, so we would have to add them https://github.com/jsdelivr/jsdelivr/blob/master/CONTRIBUTING.md
2. RF is not browser-ready by any stretch. So it would have to be bundled for the browser. 

showing what versions are in scope in the repl and maybe even letting users change versions are both good ideas, but independent of the problem of how to serve up the latest without needing to redeploy for every release.